app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -33,15 +33,28 @@ def update
       end
     end
 
+    def self.can_do_hybrid(session:)
```

Comment:
Hmmm I hear you. It's not necessarily that they want to do in person, because with Opt In they have to select that they want the remote flow. Maybe something like `opted_in_to_remote`?